Jenkinsfile

```
@@ -44,10 +44,7 @@ pipeline {
                                     environment name: 'IMAGE_TYPE', value: 'linux'
                                 }
                                 steps {
-                                    sh '''
-                                    docker buildx create --use
-                                    docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
-                                    '''
+                                    sh 'make ci-init'
```

```suggestion
                                    sh 'make buildx-init'
```

nit: the intent is to ensure buildx is initialized, so proposal to (slightly) change the target. The keyword init is good and clear 👍

---

Cf https://github.com/jenkinsci/docker-ssh-agent/pull/415#discussion_r1644102822

> WDYT of this name as it also setup `docker run --rm --privileged multiarch/qemu-user-static --reset -p yes` (so not only buildx)?

---

LGTM for `docker-init` !

---

Addressed in https://github.com/jenkinsci/docker-ssh-agent/pull/421/commits/93f56c49632fa0fa740009bf21390a6d95f76fd5